admin_bot/cogs/misc.py

```diff
@@ -89,8 +90,12 @@ async def beeReset(self, ctx):
         if await self.bot.rick(ctx):
             return
 
-        self.bee_movie_line = 0
-        await ctx.send(str("Bee movie script has been reset"))
+        if await self.bot.check_spam(ctx):
+            return
+
+        if not discord.utils.get(ctx.guild.members, name="darkstar"):
```

Comment:
Haha i like this, can you add an `else` clause in this block though to message back and say something like `await ctx.send(str("You in particular darkstar, cannot use this command."))`